ansible_files/equinix_heterogeneous_create_infra_playbook.yml

```diff
@@ -0,0 +1,38 @@
+- name: Create and configure the equinix infrastructure to run heterogneous cluster tests
+  hosts: localhost
+  vars_prompt:
+    - name: equinix_project_id
+      prompt: Equinix metal project ID in which the device(s) will be created
+      private: false
+    - name: equinix_auth_token
+      prompt: Equinix metal API token
+    - name: equinix_private_ssh_key_path
+      prompt: Path to an SSH private key allowed to connect on the created device(s)
+      private: false
+    - name: equinix_unique_id
+      prompt: Hostname prefix which will be used to identify the device(s)
+      private: false
+    - name: equinix_tf_vars_file
+      prompt: Place where the Terrafom variable file will be stored
+      private: false
+    - name: equinix_tf_state_file
+      prompt: Place where the Terrafom state file will be stored
+      private: false
+  vars:
+    terraform_equinix_workdir: "{{ [playbook_dir, '..', 'terraform_files', 'equinix-ci-machine'] | path_join | realpath }}"
+  tasks:
+    # Create the required devices to test heterogeneous scenarios
+    - ansible.builtin.import_role:
```

Comment:
just something I don't understand,
what's the difference between this syntax (with tasks and role-importing) and running it like in the other playbook?
```
  roles:
    - equinix/generic_destroy_infra
```

Reply:
it's the same meaning, when using the `roles` it will runs the roles on the `hosts` specified at playbook level. The `tasks` directive gives more control on with hosts we want to run the role.

Reply:
and in this case it will allow using only hosts generated by ``equinix/heterogeneous_create_infra`` and it will ignore ``localhost``?

Reply:
it'll be localhost exception if I specify a host, I'll do it with the libvirt role because I want to apply it on the arm machine only.

Also, I'm still discovering stuff, so it may change again :D

Reply:
ah also to pass variables, we need the `import_roles`, see below for the ipip setup.

Reply:
actually, no need for `import_task` to pass variables: https://github.com/openshift/assisted-test-infra/pull/1958/files#diff-5f56cc8fd10b4a77b4dc1326b5d54afde9b2c4b0ae285efab6a069c072505f73R23-R43

Reply:
I'll tackle that in the coming IPIP PR